packages/react/src/Link/Link.tsx

```diff
@@ -121,11 +60,11 @@ const Link = forwardRef(({as: Component = 'a', className, inline, underline, ...
   }
 
   return (
-    <StyledLink
-      as={Component}
-      className={className}
+    <Component
+      className={clsx(className, classes.Link)}
+      data-muted={props.muted}
       data-inline={inline}
-      underline={underline}
+      data-underline={underline}
       {...props}
       // @ts-ignore shh
       ref={innerRef}
```

Comment:
I probably don't need this ref anymore? @joshblack

Reply:
I think it's used in the effect for checking the ref type IIRC, let me know if I'm missing something 👀